extensions/data-transfer/portability-data-transfer-google/src/test/java/org/datatransferproject/datatransfer/google/mediaModels/GoogleMediaItemTest.java

```diff
@@ -84,81 +86,101 @@ public void getMimeType_videoModel_mimeTypeFromFilename() {
     for (Entry entry: filenameToMimeTypeMap.entrySet()) {
       videoMediaItem.setMimeType("INVALID_MIME");
       videoMediaItem.setFilename(entry.getKey().toString());
-
-      VideoModel videoModel = GoogleMediaItem.convertToVideoModel(Optional.empty(), videoMediaItem);
+      VideoModel videoModel = GoogleMediaItem.convertToVideoModel(Optional.empty(),
+          videoMediaItem);
 
       assertEquals(entry.getValue(), videoModel.getMimeType());
     }
   }
 
   @Test
-  public void getMimeType_photoModel_filenameMimeTypeIsNull() {
+  public void getMimeType_photoModel_filenameMimeTypeIsNull() throws ParseException{
     GoogleMediaItem photoMediaItem = getPhotoMediaItem();
     photoMediaItem.setFilename("file");
     photoMediaItem.setMimeType("image/webp");
-
```

Comment:
lets add this line back, we shouldn't put the test setup and the test execution next to each other.

Reply:
Done :)